### ISSUE TITLE
RSA key not Supported

### ISSUE THREAD

**Issue by primeKal** [eba-alemayehu/telebirr#1]
hello it keeps on saying value error  RSA key format is not supported when i give it a key of 1234567898765432
what should i do?

**Comment by eba-alemayehu** [comment]
hello primeKal 1234567898765432 is not correct public key format

MIIBCgKCAQEA+xGZ/wcz9ugFpP07Nspo6U17l0YhFiFpxxU4pTk3Lifz9R3zsIsu
ERwta7+fWIfxOo208ett/jhskiVodSEt3QBGh4XBipyWopKwZ93HHaDVZAALi/2A
+xTBtWdEo7XGUujKDvC2/aZKukfjpOiUI8AhLAfjmlcD/UZ1QPh0mHsglRNCmpCw
mwSXA9VNmhz+PiB+Dml4WWnKW/VHo2ujTXxq7+efMU4H2fny3Se3KYOsFPFGZ1TN
QSYlFuShWrHPtiLmUdPoP6CV2mML1tk+l7DIIqXrQhLUKDACeM5roMx0kLhUWB8P
+0uj1CNlNN4JRZlC7xFfqiMbFRU9Z4N6YwIDAQA

The correct RSA public key is some thing like this. 

Note: if your are working with telebirr they will provide you the key

**Comment by primeKal** [comment]
Yes thank you vry much, i wanted to pre try the package it until they finalize and give me the keys

**Comment by eba-alemayehu** [comment]
Sorry it is very big security  issue to give you my key... you better test
it after you got the keys
